Cargo.toml

```diff
@@ -59,3 +59,8 @@ anyhow = "1.0.32"
 thiserror = "1.0.20"
 background-jobs = " 0.8.0-alpha.2"
 reqwest = { version = "0.10", features = ["json"] }
+
+[dev-dependencies.cargo-husky]
+version = "1"
+default-features = false # Disable features which are enabled by default
+features = ["precommit-hook", "run-cargo-fmt", "run-cargo-clippy"]
```

Comment:
You'll need to run `rm -f .git/hooks/*` before running `cargo build`, so that this new husky can install its commit hooks, since the `ui/` folder is entirely removed from this codebase.